src/templates/markdown.js

```diff
@@ -225,6 +225,16 @@ class MarkdownTemplate extends React.Component {
         )
       : postNode.html
 
```

Comment:
This PR contains the commit from @leoendless , will these `.js` files influence the current branch? @leoendless

Reply:
Yes, this pr need to rebase the latest release-2.1 branch.

Reply:
@rayzhou2017 Could you please rebase it?

Reply:
rebased

Reply:
The PR should rebase branch ``release-2.1``. No need to change the javascript files.